AI-and-Analytics/Features-and-Functionality/IntelPython_XGBoost_Performance/sample.json

```diff
@@ -13,11 +13,14 @@
     {
   		"id": "idp_xgb_performance",
   		"steps": [
-         	"source /intel/oneapi/intelpython/bin/activate",
-          "python IntelPython_XGBoost_Performance.py",
-          "conda remove xgboost --y",
-          "pip install -r requirements.txt",
-          "python IntelPython_XGBoost_Performance.py"		 
+          "pip install uv",
+         	"uv venv",
+          "uv add xgboost==2.1.1",
+          "uv sync",
+          "uv run python IntelPython_XGBoost_Performance.py",
+          "uv add -r requirements.txt",
```

Comment:
I think it should be to just do uv add xgboost==0.82
I think we plan to delete the 'requirements.txt' files in future perhaps?